domains/Discussions/Controllers/AnswersUpdateController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+
+namespace Domains\Discussions\Controllers;
+
+
+use App\Http\Controllers\Controller;
+use Carbon\Carbon;
+use Domains\Discussions\Models\Answer;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class AnswersUpdateController extends Controller
+{
+    private Answer $answer;
+    private Question $question;
```

Comment:
This is not used anywhere, right?

Reply:
You're right! I can remove the question attribute from the controller.